pynestml/codegeneration/resources_spinnaker/@NEURON_NAME@_impl.h.jinja2

```diff
@@ -126,18 +132,18 @@ static void neuron_impl_do_timestep_update(
         neuron->inputs[1] = 0;
         */
 
-{%- if neuron.get_update_blocks() %}
-{%-   filter indent(2) %}
-{%-     for block in neuron.get_update_blocks() %}
-{%-       set ast = block.get_block() %}
-{%-       if ast.print_comment('*')|length > 1 %}
-/*
- {{ast.print_comment('*')}}
- */
-{%-       endif %}
-{%-       include "directives_cpp/Block.jinja2" %}
-{%-     endfor %}
-{%-   endfilter %}
+        {%- if neuron.get_update_blocks() %}
```

Comment:
C++ (generated) code indentation should be independent of template code indentation. To indent the latter, please use 
```
{%- [... indent here ...] while ... %}
```